src/audio/listeners/music/musicSongPlayNotify.ts

```diff
@@ -0,0 +1,14 @@
+import { AudioListener, NowPlayingEntry } from '#lib/audio';
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import type { MessageAcknowledgeable } from '#lib/types';
+import { resolveKey } from '@sapphire/plugin-i18next';
+
+export class UserAudioListener extends AudioListener {
+	public async run(acknowledgeable: MessageAcknowledgeable, entry: NowPlayingEntry) {
+		const requester = await this.container.client.users.fetch(entry.author).then((data) => data.username);
```

Comment:
```suggestion
		const requester = (await this.container.client.users.fetch(entry.author)).username;
```
There should never be a need for `.then` in modern async code.